compiler/testData/codegen/box/enum/enumEntriesInCompanion.kt

```diff
@@ -0,0 +1,17 @@
+// !LANGUAGE: +EnumEntries
+// TARGET_BACKEND: JVM_IR
```

Comment:
Let's use `IGNORE_BACKEND: JVM, JS` instead, so that this test will run for Native.